app/assets/javascripts/discourse/app/components/sidebar/community-section.js

```diff
@@ -31,11 +41,16 @@ export default class SidebarCommunitySection extends GlimmerComponent {
         currentUser: this.currentUser,
         appEvents: this.appEvents,
         router: this.router,
+        siteSettings: this.siteSettings,
       });
     }
   );
 
-  sectionLinks = MAIN_SECTION_LINKS.map((sectionLinkClass) => {
+  #mainSectionLinks = this.currentUser.staff
```

Comment:
Should this be `this.currentUser.admin` or can moderators get to this admin route too?

Reply:
O moderators are allowed on certain admin routes too.